### PR TITLE
WR-1179: french link blocks updates and additions

### PR DESCRIPTION
**Pull request by zetagraph**
Tickets: 

https://hive.aved.gov.bc.ca/jira/browse/WR-1179

### CODE REVIEW

File: src/config/sync/simple_block.simple_block.online_employment_services_french_link.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+uuid: 54ab1921-ad82-420e-aca4-fc8f2210caba
+langcode: en
+status: true
+dependencies:
+  config:
+    - filter.format.full_html
+id: online_employment_services_french_link
+title: 'Online Employment Services - French Link'
+content:
+  value: "<p><a data-entity-substitution=\"canonical\" data-entity-type=\"node\" data-entity-uuid=\"9627f8c2-bce6-43c0-922c-db5d4b9a36cf\" href=\"/node/71\">Services disponibles en français</a></p>\r\n"
```
Comment (infojunkie) [inline src/config/sync/simple_block.simple_block.online_employment_services_french_link.yml:10]: Any reason why this links is annotated with `data-entity-XX` and the above one in `src/config/sync/simple_block.simple_block.employment_services_french_link.yml` is not?

Comment (zetagraph) [reply src/config/sync/simple_block.simple_block.online_employment_services_french_link.yml:10]: I think the drupal link autocomplete module converts paths to node IDs and adds these attributes. I've checked the links and paths and I think they are fine but I will double check things tomorrow. 

Screen Shot 2023-01-23 at 9 56 21 PM

Comment (infojunkie) [reply src/config/sync/simple_block.simple_block.online_employment_services_french_link.yml:10]: Thanks. Would be great to keep the node number out of the block and only refer to the page via its path like the other blocks in this PR.

Comment (zetagraph) [reply src/config/sync/simple_block.simple_block.online_employment_services_french_link.yml]: Should be updated now.

Comment (infojunkie) [reply src/config/sync/simple_block.simple_block.online_employment_services_french_link.yml]: Thanks, lgtm

**Review by infojunkie — APPROVED** [review]
Lgtm, thanks

**Pull request merged**